zip-0230.rst

```diff
@@ -317,26 +315,48 @@ An issuance action, ``IssueAction``, is the instance of issuing a specific Custo
 |                             |                          |                                           | * :math:`\mathsf{finalize}`                                         |
 |                             |                          |                                           | * The remaining bits are set to :math:`0\!`.                        |
 +-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
 
-The encodings of each of these elements are defined in ZIP 227 [#zip-0227]_.
+The encoding of ``IssueNote`` is described below.
 Note that we allow the number of notes (represented by ``nNotes``) to be zero. 
 This allows for issuers to create Issuance Actions to only finalize an issued Asset, without needing them to simultaneously issue more of that Asset. 
 
+Issue Note Description (``IssueNote``)
+--------------------------------------
+
+An issuance note, ``IssueNote`` contains the following fields:
+
++-----------------------------+--------------------------+--------------------------------------+---------------------------------------------------------------+
+| Bytes                       | Name                     | Data Type                            | Description                                                   |
++=============================+==========================+======================================+===============================================================+
+|``43``                       |``recipient``             |``byte[43]``                          |The raw encoding of an Orchard Raw Payment Address, as         |
+|                             |                          |                                      |per protocol §5.6.4.2 ‘Orchard Raw Payment Addresses’.         |
++-----------------------------+--------------------------+--------------------------------------+---------------------------------------------------------------+
+|``8``                        |``value``                 |``uint64``                            |The amount of the Asset being issued in this note.             |
++-----------------------------+--------------------------+--------------------------------------+---------------------------------------------------------------+
+|``32``                       |``assetBase``             |``byte[32]``                          |The encoding of the Asset Base                                 |
+|                             |                          |                                      |:math:`\mathsf{AssetBase^{Orchard}}\!`.                        |
++-----------------------------+--------------------------+--------------------------------------+---------------------------------------------------------------+
+|``32``                       |``rho``                   |``byte[32]``                          |This is defined and encoded in the same manner as for          |
```

Comment:
need to be specific about where exactly it is defined or define it here. (x2)

Reply:
Added that in all the rows it was missing.